File: src/IsbInterop.CSharp/Tasks/Wrappers/JobInfo.cs
namespace IsbInterop.Tasks.Wrappers
{
  /// <summary>
  /// Обертка над IJobInfo.
  /// </summary>
  internal class JobInfo : CustomJobInfo, IJobInfo
  {
     /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwIJobInfo">COM-объект IJobInfo.</param>
    public JobInfo(object rcwIJobInfo) : base(rcwIJobInfo) { }
  }
}

File: src/IsbInterop/Tasks/Wrappers/JobInfo.vb
Namespace Tasks.Wrappers
  ''' <summary>
  ''' Обертка над IJobInfo.
  ''' </summary>
  Friend Class JobInfo
    Inherits CustomJobInfo
    Implements IJobInfo
    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwIJobInfo">COM-объект IJobInfo.</param>
    Public Sub New(rcwIJobInfo As Object)
      MyBase.New(rcwIJobInfo)
    End Sub
  End Class
End Namespace